app/db/models/lawpolicy/event.py

```diff
@@ -0,0 +1,70 @@
+# TODO: Implement events
```

Comment:
We could remove this file completely & move the code to either a ticket/Notion?

Reply:
Happy to

Reply:
I mention it because there seems to be some uncertainty around the status of Events at the moment, and the structure could well change when Alan is back...

Reply:
See https://linear.app/climate-policy-radar/issue/BAK-1303/decide-how-events-will-be-implemented-in-new-dfc-world